release/src/main/groovy/TestScripts.groovy

```diff
@@ -95,16 +116,31 @@ class TestScripts {
      }
    }
 
-   // Check for expected results in stdout of the last command
+   // Check for expected results in stdout of the last command, if fails, log errors then exit.
    public void see(String expected) {
      if (!var.lastText.contains(expected)) {
        var.startDir.deleteDir()
        println "Cannot find ${expected} in ${var.lastText}"
-       _error("Cannot find expected text")
+       error("Cannot find expected text")
      }
      println "Verified $expected"
    }
 
+   public boolean seeOneOf(List<String> expected) {
+     for (String expect: expected) {
+       if(var.lastText.contains(expect)) {
+         println "Verified $expect"
+         return true
+       }
+     }
+     println "Cannot find ${expected} in text"
+     return false
+   }
+
+   public boolean seeExact(String expected) {
```

Comment:
Do we use this anywhere?

Even if we do, it looks more like "seeSubstring"

Reply:
yes, we do need it while checking whether tables are created/delete from dataset. 
Renamed method being seeSubstring()